src/utils/components/DiskModal/DiskFormFields/StorageClassSelect.tsx

```diff
@@ -45,6 +45,12 @@ const StorageClassSelect: React.FC<StorageClassSelectProps> = ({
     [storageClasses],
   );
 
+  useEffect(() => {
+    if (!storageClass) {
```

Comment:
```suggestion
    if (!storageClass && loaded) {
```
As the continuous change of storageClasses during load can create some ui flickering probably